docs/data-apps/index.md

```diff
@@ -26,12 +26,36 @@ import DocCardList from '@theme/DocCardList';
 
 ## Accessing the Applications
 
-You can find the Data Applications in the `Applications` tab in the left sidebar of your Snowplow Console. If the status is not `Live`, you can click on the tile and request access. A Snowplow Customer Success Manager will then get in contact with you to discuss the Snowplow Digital Analytics package.
+You can find the Data Applications in the `Applications` tab in the left sidebar of your Snowplow Console. An application can be in one of three states:
+* _By request_ — If you have not yet purchased a package that includes the application, you can click on “Learn more” and register your interest. A Snowplow Customer Success Manager will then get in contact with you to discuss getting access to the application.
+* _Available_ — The application is ready to be set up. See the [installation instructions](#installing-applications) below.
+* _Live_ — The application is ready to use.
 
 Once the application is installed, clicking on the tile will launch the application in a separate browser tab. By default, anyone in your Console organization will be able to access data applications.
 
 If you wish to invite others to use data applications but not have access to the rest of Console, you can [create a new user](/docs/using-the-snowplow-console/managing-users/adding-removing/index.md) and add a user with the `Data applications user` role. That user will then only see the Data Applications tab within Console. This permissions can be managed in the [usual way](/docs/using-the-snowplow-console/managing-users/managing-permissions/index.md).
 
+## Installing applications
+
+Once you have purchased an application you are able to install it. Click the tile to begin installing the application and follow the steps. 
+
+![Image of application tile showing start install process](images/install-app-tile.png)
+
+During installation you will need to [provide a connection](#application-connections) to the warehouse you would like the application to use. It will also highlight any pipeline and data model dependencies that are required.
+
+![Image of application install process](images/sample-app-install.png)
+
+## Dependencies on data models
+
+Generally applications will depend on data models. If there are dependencies then the installation flow will highlight which models are required and what models you currently have running through console. It will also highlight any properties that need to be enabled on these data models. If you are running these models yourself outside of console then you will need to manually check the requirements for each application. These requirements are highlighted within the documentation pages for each application.
```

Comment:
```suggestion
Generally, applications will depend on data models. If there are dependencies, then the installation flow will highlight which models are required and what models you currently have [running via BDP](/docs/modeling-your-data/running-data-models-via-snowplow-bdp/standard-models/index.md). It will also highlight any properties that you need to enable or configure for these data models.

:::note
If you are running the necessary data models yourself outside of BDP, then you will need to manually check that your setup satisfies the requirements for each application. These requirements are listed within the documentation pages for each application.
:::
```